my-react-app/src/app/components/baseComponents/Game/utils.js

```diff
@@ -0,0 +1,13 @@
+const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+
+function calculateWinner(squares) {
+  for (let i = 0; i < lines.length; i++) {
```

Comment:
Replace this `for` with an [array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) method. Look into the `.find()`method